<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwApi</class>
 <widget class="QMainWindow" name="qwApi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Api z Api</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QTableWidget */
QTableWidget#twDane{
	background-color: white;
	color: #595959;
	alternate-background-color: #f0f0f0;
	selection-background-color: #dddddd;
	selection-color: #f16e00;
  	border-radius: 5px;
	border: none;
	font-size: 8;
	gridline-color: #dddddd;
    padding: 10px 10px;
	font: 8pt &quot;Helvetica 55 Roman&quot;;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QTableWidget#twDane QHeaderView::section {
	font-weight: bold;
	color: #595959;
	background-color: white;
	border: none;
	font-size: 8pt;
}

QTableWidget#twDane QHeaderView::section:hover {
	background-color: white;
	color: black;
}

QTableWidget#twDane QTableView::item:hover {
	background-color: white;
	color: black;
}


/* QPushButton */
QPushButton {
	background-color: #7fc5b7;
	border: none;
	color: white;
	font-weight: bold;
	border-radius: 0px;
 	text-align: left;
    padding: 10px 50px;
	font: 10pt &quot;Helvetica 55 Roman&quot;;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}

QPushButton:hover {
	background-color: white;
	color: #ff7606;
}

QPushButton:pressed {
	background-color: white;
	color: #ff7606;
}

/* QFrame */
QFrame {
	font: 10pt &quot;Helvetica 55 Roman&quot;;
    border-top-left-radius: 10px;
    border-bottom-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 43, 43);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignLeft">
     <widget class="QWidget" name="wgtLeftMenu" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="wgtLeftSubMenu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frameTop">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pbPM10">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}

</string>
               </property>
               <property name="text">
                <string>PM10</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbNO2">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}
</string>
               </property>
               <property name="text">
                <string>NO2</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSO2">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}
</string>
               </property>
               <property name="text">
                <string>SO2</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbO3">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}
</string>
               </property>
               <property name="text">
                <string>O3</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QFrame" name="frameBottom">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="pbDane">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(149, 186, 231);
}

QPushButton:checked {
   background-color: rgb(0, 88, 113);
}
</string>
               </property>
               <property name="text">
                <string>Dane</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgtCenterMenu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(149, 186, 231);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="twDane">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableView::item:selected{
background-color : #bde0ff; 
selection-color : #000000;
}
color: black;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>true</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::DotLine</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Stacja</string>
             </property>
             <property name="foreground">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Miasto</string>
             </property>
             <property name="foreground">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wskaźnik</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wgtMainBody" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QWidget" name="wgtMenuTop" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frameTopMenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pbPM10_3">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}
</string>
               </property>
               <property name="text">
                <string>Interpolacja</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbPM10_2">
               <property name="styleSheet">
                <string notr="true">QPushButton {
   background-color: rgb(127, 197, 183);
}

QPushButton:checked {
   background-color: rgb(45, 90, 75);
}
</string>
               </property>
               <property name="text">
                <string>Warstwice</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(127, 197, 183);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="leftMargin">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QgsMapCanvas" name="graphicsView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapCanvas</class>
   <extends>QGraphicsView</extends>
   <header>qgsmapcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
